api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -409,6 +409,46 @@ func (in *DriverRegistrarSpec) DeepCopy() *DriverRegistrarSpec {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *EtcdAuthClientSpec) DeepCopyInto(out *EtcdAuthClientSpec) {
+	*out = *in
+	if in.EnableAuthentication != nil {
+		in, out := &in.EnableAuthentication, &out.EnableAuthentication
+		*out = new(bool)
+		**out = **in
```

Comment:
is in,out here a local var instead of   (in * EtcdAuthClientSpec) and (out *EtcdAuthClientSpec) in the function?
can u change the local var to sth else if thats the case?

Reply:
resolved as its autogenerated code